starport/templates/app/stargate/deploy/deploy.yaml.plush

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
can we use `akash` folder instead of `deploy`?

Reply:
Sure